tutorial/katacoda-scenarios/flyte-install/step3.md

```diff
@@ -0,0 +1,3 @@
+Register Workflow
+
+`docker run --network host -e FLYTE_PLATFORM_URL='[[HOST_IP]]:30081' lyft/flytesnacks:b347efa300832f96d6cc0900a2aa6fbf6aad98da pyflyte -p flytesnacks -d development -c sandbox.config register workflows`{{execute HOST2}}
```

Comment:
@kumare3 Step 4: May be link to UI or flyte-cli command to query the status of the workflow?

Reply:
That's a great idea

Reply:
Currently we don't have flyte-cli in host machine